include/accessor.h

```diff
@@ -594,6 +614,10 @@ class local_accessor {
 	using const_reference = const DataT&;
 	using size_type = size_t;
 
+	CELERITY_DETAIL_HACK_CLANG_TIDY_ALLOW_NON_CONST(local_accessor)
+	local_accessor(local_accessor&&) noexcept = default;
+	local_accessor& operator=(local_accessor&&) noexcept = default;
```

Comment:
I believe neither this move-assignment nor the existing copy-assignment definitions are actually needed.